Template10 (Library)/Controls/HamburgerButtonInfo.cs

```diff
@@ -92,7 +92,14 @@ public double MaxWidth
             set { Set(ref _MaxWidth, value); }
         }
 
-        public override string ToString() => string.Format("{0}({1})", PageType?.ToString() ?? "null", PageParameter?.ToString() ?? "null");
+		string _navigationGroup;
```

Comment:
I think the difference comes from "Keep tabs" which is true in my settings, not from tab size (which is 4).